.github/workflows/spelling.yml

```diff
@@ -0,0 +1,11 @@
+name: codespell
+on: [pull_request, push]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install codespell
+      - run: codespell --ignore-words-list="alls, all's"
```

Comment:
I couldn't find "alls" and "all's" in our codebase. They aren't correct grammar anyway so they should be removed.

Reply:
> I couldn't find "alls" and "all's" in our codebase.

`.github/workflows/sort_alls.yaml`

Reply:
Pretty sure sigma did that 💀 
Someone should fix it

Reply:
I just did all’s just Incase someone used it

Reply:
> Pretty sure sigma did that skull
Someone should fix it

It was already explained that it is correct since it refers to `__all__`s

Reply:
I was about to say I realized that lol 💀